sdk/samples/iot/README.md

```diff
@@ -100,15 +100,15 @@ This section provides an overview of the different samples available to run and
 
 - *Executable:* `paho_iot_hub_pnp_sample`
 
-  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_sample.c) connects an IoT Plug and Play enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect.  X509 self-certification is used.
+  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_sample.c) connects an IoT Plug and Play enabled device (a thermostat) with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will continue. If 3 timeouts occur consecutively, the sample will disconnect. X509 self-certification is used.
 
   To interact with this sample, **you must use the Azure IoT Explorer**. The capabilities are listed below:
 
-- **Device Twin**: Two device twin properties are supported in this sample.
+- **Device Twin**: Two device twin properties are supported in this sample:
   - A desired property named `targetTemperature` with a `double` value for the desired temperature.
-  - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature.
+  - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature reached since boot.
```

Comment:
```suggestion
  - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature reached since device boot.
```

Reply:
also, add a blank line after this.